core/src/main/java/org/apache/accumulo/core/util/Pair.java

```diff
@@ -23,8 +23,8 @@
 import java.util.Objects;
 
 public class Pair<A,B> {
-  A first;
-  B second;
+  final A first;
+  final B second;
```

Comment:
Do you know why these are visible? They could probably be private, too.

Reply:
Seems they can be protected, a subclass uses them.  I made the change in e78ab35

Reply:
The subclass should probably be made to call `.getFirst()` and `.getSecond()` instead. It doesn't really matter, I guess. Pair is not public API.

Reply:
Updated in 62efb44